pom.xml

```diff
@@ -53,6 +53,7 @@ THE SOFTWARE.
     <module>bom</module>
     <module>websocket/spi</module>
     <module>websocket/jetty10</module>
+    <module>websocket/jetty12</module>
```

Comment:
For consistency with the existing `jetty10` module, the addition of this new module will require a corresponding change to `.idea/encodings.xml`:

```
    <file url="file://$PROJECT_DIR$/websocket/jetty12/src/filter/resources" charset="UTF-8" />
    <file url="file://$PROJECT_DIR$/websocket/jetty12/src/main/java" charset="UTF-8" />
    <file url="file://$PROJECT_DIR$/websocket/jetty12/src/main/resources" charset="UTF-8" />
```

Reply:
TBH, having those ide files here is a really big pain. 
As I have winstone as a module in the same project,  some lines are added for winstone as well in this file.